tests/unit-test.js

```diff
@@ -181,3 +182,24 @@ describe('Util:', () => {
     });
   });
 });
+
+describe('Index', async () => {
+  fixture.writeSync('tests', {
+    'add.js':`exports.add = (a, b) => {
+      return a + b;
+    }`
+  });
+  describe('runTest', () => {
+    it('test passes', async (assert) => {
+      let done = assert.async();
+      assert.ok(await runTest({ codeArray: ['add(3,4) // equals: 7;'], importsArray: ['import { add } from \'./tests/add\''] }));
+      done();
+    });
+    it('test fails with error', async (assert) => {
```

Comment:
its strange to mix async functions + assert.rejects, you should likely choose one or the other

Reply:
Removed `done`